source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst

```diff
@@ -33,34 +33,34 @@ Locate the following control system components and tools
       - 2x PWM y-cables
       - 120A Circuit breaker
       - 4x 40A Circuit breaker
-      - 6 AWG Red wire
-      - 10 AWG Red/Black wire
-      - 18 AWG Red/Black wire
-      - 22AWG yellow/green twisted CAN cable
-      - 16x 10-12 AWG  (yellow) ring terminals
+      - 6 AWG (16 :math:`mm^2`) Red wire
+      - 10 AWG (6 :math:`mm^2`) Red/Black wire
+      - 18 AWG (1 :math:`mm^2`) Red/Black wire
+      - 22AWG (0.5 :math:`mm^2`) Yellow/Green twisted CAN cable
+      - 16x 10-12 AWG (4 - 6 :math:`mm^2`)(Yellow) ring terminals
```

Comment:
I think that most teams use AWG regardless of location, so I'm not sure that it needs metric equivalents.
Also, fix the spacing on `22AWG`, and between the two parentheses on the ring terminals line.

Reply:
I agree that FRC rules are written in AWG, but this issue came about because the translation teams asked for it specifically around wire size.  Space fixed.